test/test_podman_baseline.sh

```diff
@@ -437,7 +437,7 @@ fi
 ########
 FILE=./Dockerfile
 /bin/cat <<EOM >$FILE
-FROM docker/whalesay:latest
+FROM pharshal/whalesay:latest
```

Comment:
@baude do you have a spot on quay.io set aside for other Podman images that we could tuck away whalesay too?

Reply:
we do have quay.io/libpod but quay isnt multiarch yet I believe? so I dont think that helps here.  Also, @harche are you committing to keeping this image current and rebuilt at the same cadence as the whalesay image?

Reply:
@baude `pharshal/whalesay:latest` is a multi-arch image. For x64 it refers to the old `docker/whalesay:latest` so there is nothing to maintain there. But for `ppc64le` it refers to `pharshal/whalesay_ppc64le` which I can maintain.

Reply:
Exactly, and i am asking if you are prepared/committed to maintain that image perpetually.

Reply:
@baude sure, I can do that.